### ISSUE TITLE
Don't ovverride settings just because you looked at it

### ISSUE THREAD

**Issue by jenny787** [Smurf-IV/Elucidate#84]
When using Elucidate with Drivepool, best practice is to place the content outside the garbled drivepool directory.  The only way to do this is to modify the snapraid config file manually with a text editor.  However, once you load this file into Elucidate, it will overwrite the content file setting and place it automatically inside the drivepool directory.  So then, to fix it, i have to save the settings, then go back to the text editor to fix it, save it from the text editor.  But then, make sure to not open settings, because every time you do, you will have to go back to the text editor to fix it.

So it would be nice in the Elucidate settings gui to either be able to specify such content file placement or not have it automatically overwrite it.

**Comment by Smurf-IV** [comment]
@jenny787 
Thanks for the report. 
1) Can you give an example config with the setting you are on about please ?
2) And what version of Elucidate you are using ?

Thanks

**Comment by jenny787** [comment]
I was using the latest version at the time.  
the main setting I am speaking of is the disk mounting lines in the config.  elcuidate's gui allows you to drag-drop the disks in.  But the problem is when you want the disk mounted not from the root, but from a subfolder.  A lot of drive pooling setups do this.  So I configure the ini with the subfolders in the mounts, but if i open that file in elcuidate's gui, when it is verifying all the disks, it replaces the subfolders with the root folder.  So then, I have to close the gui and open the ini file with notepad and fix it all.  As long as i don't open that part of the gui, it's all good.

**Comment by jenny787** [comment]
I wrote an article for this and how I deal with it.  I will copy it here also.  But it would be great if you could fix this issue.


Be Careful Opening the Settings File

Elucidate will change your file paths without warning if you open the settings icon and press the save button.
Do Not Ever Save the Settings File in the GUI

When using Drivepool, best practice is to place the snapraid content files OUTSIDE the PoolPart folder.  The only way to do this with Elucidate as the GUI is the first change those paths using a text editor.  You cannot do it in the GUI.  Not only that, but you have to be careful to never EVER save the file in the GUI.


But I have to Save it at least ONCE, right?

Unfortunately, you have to set the file at least the first time.  So how do you do this?  Follow the steps:

    First, set everything properly using a text editor.
    Open this file in the GUI Settings.  You will have to save it, and it will mess up the file paths.
    Open the same file back in the text editor, and fix those paths back.
    Now, everything should be OK.  You can open the settings to check, but do not save it.
    If you ever need to change anything in the settings, you need to repeat all of this.

**Comment by Smurf-IV** [comment]
Thanks for the steps.
I'll investigate the next time I look at the code.

Just to be clear, is this just when using _drivepool_, or does it "Mess up" other settings as well ?

**Comment by jenny787** [comment]
No, this would not be just for drivepool.  The root of the problem is that Elucidate assumes the content files will be in the root of whatever the disk roots are showing.  but with the text file you can place the content file wherever you want.  So if you press the save button, Elucidate will change the path to the root if it is not there already.
